CredentialProvider/CredentialList.storyboard

```diff
@@ -0,0 +1,643 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the first storyboard we would add into this project. I'm not strongly against storyboard but it seems like the convention has been programatic layout and would like to keep things consistent. I can hop in and help convert this to programatic views if needed 😀.

Reply:
I assume `CredentialProvider/MainInterface.storyboard` is required because it is the entry point that is referenced in the extension's `Info.plist` but the others could probably be turned into programmatic views.